fastcrypto-vdf/benches/class_group.rs

```diff
@@ -43,18 +41,15 @@ fn class_group_ops(c: &mut Criterion) {
 }
 
 fn qf_from_seed_single<M: Measurement>(discriminant_string: &str, group: &mut BenchmarkGroup<M>) {
-    for k in [1, 2, 4, 8, 16, 32, 64] {
-        let discriminant =
-            Discriminant::try_from(BigInt::from_str_radix(discriminant_string, 10).unwrap())
-                .unwrap();
+    let discriminant =
+        Discriminant::try_from(BigInt::from_str_radix(discriminant_string, 10).unwrap()).unwrap();
```

Comment:
ooc, when can this panic?

Reply:
Due to the unwrap, this happens if the given discriminant is not a negative (probable) prime that is 1 mod 8.